presto-docs/src/main/sphinx/release/release-0.199.rst

```diff
@@ -0,0 +1,50 @@
+=============
+Release 0.199
+=============
+
+General Changes
+---------------
+
+* Add support for spatial LEFT join.
+* Add :func:`hmac_md5`, :func:`hmac_sha1`, :func:`hmac_sha256`, and :func:`hmac_sha512` functions.
+* Add :func:`array_sort` function that takes a lambda as a comparator.
+* Add :func:`line_locate_point` geospatial function.
+* Add support for authentication with JWT access token.
+* Add support for ``ORDER BY`` clause in aggregations for queries that use grouping sets.
+* Expand grouped execution support to ``GROUP BY`` and ``UNION ALL``, making it possible
+  to execute aggregations with less peak memory usage.
+* Change the signature of ``round(x, d)`` and ``truncate(x, d)`` functions so that
+  ``d`` is of type ``INTEGER``. Previously, ``d`` could be of type ``BIGINT``.
+  This behavior can be reverted with the ``deprecated.legacy-round-n-bigint`` config option
+  or the ``legacy_round_n_bigint`` session property.
+* Add support for installing Presto server RPM to machines with OpenJDK.
+* Optimize the :func:`ST_Intersection` function for rectangles aligned with coordinate axes
+  (e.g., polygons produced by the :func:`ST_Envelope` and :func:`bing_tile_polygon` functions).
+* Finish a query early if the table on the right side of an INNER or RIGHT join doesn't produce any rows. Similarly,
```

Comment:
Lowercase these words

Reply:
This is not accurate, since it really means "join and not "query". As written, it's only true for certain simple cases. How about
```
* Finish joins early when possible if one side has no rows. This happens for
  either side of an inner join, for the left side of a left join, and for the
  right side of a right join.